### DISCUSSION TITLE
黑塔

### DISCUSSION THREAD

**Discussion by yansteve454** [share121/inter-knot#2780] [其他]
Uploading Video-22-33-21.mp4…

好看的黑塔

**Comment by shan-dgha** [comment]
看看黑塔<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火礼物" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火礼物" width="75" >黑塔真好看
